systemservices/marketplace/mesg.yml

```diff
@@ -36,41 +36,110 @@ tasks:
                     type: String
                   manifestProtocol: &manifestProtocol
                     type: String
-                  manifestData: &manifestData
+                  createTime: *createTime
+              offers: &offers
+                type: Object
+                repeated: true
+                optional: true
+                object: &offer
+                  offerIndex: &offerIndex
+                    type: String
+                  price: &price
+                    type: String
+                  duration: &duration
+                    type: String
+                  active:
+                    type: Boolean
+                  createTime: *createTime
+              purchases: &purchases
+                type: Object
+                repeated: true
+                optional: true
+                object: &purchase
+                  purchaser: &purchaser
+                    type: String
+                  expire: &expire
+                    type: String
+                  createTime: *createTime
+      error: &outputError
+        name: "Error"
+        description: "Output when an error occurs"
+        data:
+          message:
+            name: "Message"
+            description: "The error message"
+            type: String
+          code:
+            name: "Error code"
+            description: "The error code"
+            type: String
+            optional: true
+  getService:
+    inputs:
+      sid: *sid
+    outputs:
+      error: *outputError
+      success:
+        data:
+          sid: *sid
+          createTime: *createTime
+          owner: *owner
+          offers: *offers
+          purchases: *purchases
+          versions:
+            type: Object
+            repeated: true
+            optional: true
+            object:
+              <<: *version
+              manifestData: &manifestData
+                type: Object
+                optional: true
+                object:
+                  version:
+                    type: String
+                  service: &manifestServiceData	
                     type: Object
-                    optional: true
                     object:
-                      version:
+                      hashVersion:
```

Comment:
this one was supposed to be in definition and needs to be removed anyway based on https://github.com/mesg-foundation/core/pull/906